Healthy/Classes/Generated/UIImage.Generated.swift

```diff
@@ -9,4 +9,12 @@ extension UIImage {
     static var accentColor: UIImage {
       UIImage(named: "AccentColor")!
     }
+
+    static var icCheckboxNotSelected: UIImage {
+      UIImage(named: "ic_checkbox_not_selected")!
+    }
+
+    static var icCheckboxSelected: UIImage {
+      UIImage(named: "ic_checkbox_selected")!
```

Comment:
```suggestion
      UIImage(named: "icon_checkbox_not_selected")!
    }

    static var icCheckboxSelected: UIImage {
      UIImage(named: "icon_checkbox_selected")!
```

To follow the swift way. Please make sure to change the asset name.